Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -389,7 +393,7 @@ instance : SMul S (QuadraticMap R M N) :=
         letI := SMulCommClass.symm S R N
         ⟨a • B, by simp [h]⟩ }⟩
 
-@[simp]
+@[simp, norm_cast]
```

Comment:
Naive question: is this the right direction for a norm_cast lemma? For example, I'd expect norm_cast to normalise `a • ⇑Q` to `⇑(a • Q)` --- which is the opposite direction of the lemma. (Is there some documentation on the order of norm_cast lemmas? I can well imagine this is fine, and I just haven't found the right docs yet.)

Reply:
I think norm_cast only accepts one direction anyway, and the tactic applies the lemma in reverse.

Reply:
:bulb: